requirements.yml

```diff
@@ -1,5 +1,7 @@
 ---
 collections:
+  - name: community.aws
+    version: 3.1.0
```

Comment:
move it under `# molecule ec2 tests`

the idea was to keep requirements organized somehow